.gitignore

```diff
@@ -35,3 +35,10 @@ jspm_packages
 
 # Optional REPL history
 .node_repl_history
+
+.DS_Store
+patternlab.json
```

Comment:
I'm not sure what that is but `patternlab.json` doesn't exist so I removed that line from `.gitignore`